kvmagent/kvmagent/plugins/vm_plugin.py

```
@@ -6918,6 +6926,12 @@ def make_pci_device(pciDevices):
                         raise kvmagent.KvmError('failed to /usr/lib/nvidia/sriov-manage -d %s: %s, %s' % (addr, o, stderr))
 
 
+                try:
+                    check_device_in_use(addr)
+                except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)